docs/static/common/icons.svg

```diff
@@ -431,9 +431,9 @@
     </symbol>
     <symbol id="icon-star-dynamic" viewBox="0 0 22 22">
         <!-- full star -->
-        <path d="M11.0418 2.19068C11.3405 1.26977 12.6556 1.26977 12.9544 2.19068L15.0869 9H21.998C22.9684 9 23.3684 9.97367 22.5863 10.5428L16.9857 14.6188L19.1191 21.1958C19.4171 22.1146 18.3532 22.8737 17.5682 22.3024L11.9981 18.2486L6.42795 22.3024C5.64292 22.8737 4.57902 22.1146 4.87705 21.1958L7.01047 14.6188L1.40983 10.5428C0.627778 9.97367 1.04044 9 2.0108 9H8.92191L11.0418 2.19068Z" style="fill: var(--color-star-rating-full);" />
+        <path d="M11.0418 2.19068C11.3405 1.26977 12.6556 1.26977 12.9544 2.19068L15.0869 9H21.998C22.9684 9 23.3684 9.97367 22.5863 10.5428L16.9857 14.6188L19.1191 21.1958C19.4171 22.1146 18.3532 22.8737 17.5682 22.3024L11.9981 18.2486L6.42795 22.3024C5.64292 22.8737 4.57902 22.1146 4.87705 21.1958L7.01047 14.6188L1.40983 10.5428C0.627778 9.97367 1.04044 9 2.0108 9H8.92191L11.0418 2.19068Z" fill="var(--color-star-rating-full, #ccc)" />
         <!-- full star/half star clipped -->
-        <path d="M11.0418 2.19068C11.3405 1.26977 12.6556 1.26977 12.9544 2.19068L15.0869 9H21.998C22.9684 9 23.3684 9.97367 22.5863 10.5428L16.9857 14.6188L19.1191 21.1958C19.4171 22.1146 18.3532 22.8737 17.5682 22.3024L11.9981 18.2486L6.42795 22.3024C5.64292 22.8737 4.57902 22.1146 4.87705 21.1958L7.01047 14.6188L1.40983 10.5428C0.627778 9.97367 1.04044 9 2.0108 9H8.92191L11.0418 2.19068Z" style="fill: var(--color-star-rating-half); clip-path: var(--half-star-clip);" />
+        <path d="M11.0418 2.19068C11.3405 1.26977 12.6556 1.26977 12.9544 2.19068L15.0869 9H21.998C22.9684 9 23.3684 9.97367 22.5863 10.5428L16.9857 14.6188L19.1191 21.1958C19.4171 22.1146 18.3532 22.8737 17.5682 22.3024L11.9981 18.2486L6.42795 22.3024C5.64292 22.8737 4.57902 22.1146 4.87705 21.1958L7.01047 14.6188L1.40983 10.5428C0.627778 9.97367 1.04044 9 2.0108 9H8.92191L11.0418 2.19068Z" fill="var(--color-star-rating-half, transparent)" />
```

Comment:
Removing the clipping would break half stars; not sure how those would still work for you.

Reply:
Yep, I'm seeing this issue:

![Screen Shot 2022-03-31 at 11 08 09 AM](https://user-images.githubusercontent.com/38065/161121885-e5b5d117-05ae-4f4e-a8ec-8e5e4929b30e.png)